README.md

```diff
@@ -13,9 +13,9 @@ applications' configurations, leveraging easy application integration.
 
 ## Often used links ##
 
-- [build server](http://build.libelektra.org:8080/)
-- [tutorials](http://git.libelektra.org/blob/master/doc/tutorials/)
-- [API docu](http://doc.libelektra.org/api/latest/html/)
+- [build server](https://build.libelektra.org/)
```

Comment:
> I just created a new branch based on the last good commit of super-ls, so no cherry picking needed.

Thank you for providing the new branch. Currently the code in `kdb-complete` does not compile on my machine. It works after I remove the files`superls.cpp` and `superls.hpp`, and replace 

```cpp
#include <superls.hpp>
```

in `complete.cpp` with

```cpp
#include <complete.hpp>
```

.